components/bl-cascadeSelect-component/README.md

```diff
@@ -0,0 +1,39 @@
+# Cascade Select
+
+Cascade Select is a component of Backendless UI-Builder designer. This allows you to select a value from a nested structure of options.
+
+The component based on external [Cascade Select](https://www.primefaces.org/primereact/cascadeselect/).
+
+## Properties
+
+| Property          | Type    | Default Value       | Logic                       | Data Binding | UI Setting | Description                                                                                                                                                              |
+|-------------------|---------|---------------------|-----------------------------|--------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cascade           | JSON    |                     | Cascade Logic               | YES          | YES        | Allows determinate an array of select items to display as the available options. Watch [Codeless Examples](#Examples). Signature of polygon: `[{name, code, ?children}]` |
+| Placeholder       | String  |                     | Placeholder  Logic          | YES          | YES        | Allows determinate placeholder for input                                                                                                                                 |
+
+## Events
+
+| Name          | Triggers                      | Context Blocks                                               |
+|---------------|-------------------------------|--------------------------------------------------------------|
+| On Click Item | when the user select the item | Item: `{name: String, code: String, levelOfNesting: Number}` |
+
+## Action
+
+| Action        | Inputs | Return                   |
+|---------------|--------|--------------------------|
+| Get Select in |        | Object: of a select item |
```

Comment:
How about add an actions same as for default select?
Set Options
Get Options
Set Value
Get Value

Reply:
please describe these actions here in the doc